### PR TITLE
[0.13] Add release notes for wallet/mempool rejections. (PR #9302 and #9290)

### PR DESCRIPTION
**Pull request by gmaxwell**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
ACK 3304cb7

File: doc/release-notes.md [hunk 1]
```diff
@@ -42,8 +42,23 @@ but severe issues with the libc++ version on 10.7.x keep it from running reliabl
...
+-----------------------------------------------------------------
+
+When a newly created transaction failed to enter the mempool due to
+the limits on chains of unconfirmed transactions the sending RPC
+calls would return an error.  The transaction would still be queued
+in the wallet and, once some of the parent transactions were
+confirmed, broadcast after the software was restarted.
+
+This behavior has been changed to return success and to reattempt
+mempool insertion at the same times transaction rebroadcast is
```
Comment (instagibbs) [inline doc/release-notes.md]: uber nit: /times/time

**Comment by instagibbs** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9323/commits/3304cb75f05a65b50050adaf63fde8b72eb3dc9e

**Review by jonasschnelli — APPROVED** [review]
ACK

**Comment by morcos** [comment]
Is it worth mentioning that this also applies to transactions created with the GUI?

**Comment by gmaxwell** [comment]
@morcos I changed the title of the section to make it more generic, I think that covers it. (otherwise, if you'd like to suggest language...)

**Comment by morcos** [comment]
ACK 57aec3b

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/commit/57aec3b0b5993568571c352643abb6f3d79ffdd7

**Pull request merged**